### ISSUE TITLE
confusion about the implementation

### ISSUE THREAD

**Issue by yuxiaokang-source** [RinneSz/GMAE#3]
I have some confusion about the implementation of GMAE ,for example in the function `process_samples` 
```
  def process_samples(self, batch_size, n_id, adj):
      edge_index = adj[0].edge_index
      if edge_index.size(1) != 0:
          edge_index = to_undirected(edge_index)
      n_nodes = len(n_id)
      edge_sp_adj = torch.sparse.FloatTensor(edge_index,
                                             torch.ones(edge_index.shape[1]),
                                             [n_nodes, n_nodes])
      edge_adj = edge_sp_adj
      return [self.dataset.x[n_id], self.dataset.y[n_id[0]], edge_adj]
```
and 
```
def preprocess_item(item):
    x, y, adj = item[0], item[1], item[2].to_dense()
    N = x.size(0)

    # node adj matrix [N, N] bool
    adj = adj.bool()

    shortest_path_result, path = algos.floyd_warshall(adj.numpy())
    spatial_pos = torch.from_numpy((shortest_path_result)).long()
    attn_bias = torch.zeros([N, N], dtype=torch.float)

    in_degree = adj.long().sum(dim=1).view(-1)
    out_degree = adj.long().sum(dim=0).view(-1)
    return x, y, adj, attn_bias, spatial_pos, in_degree, out_degree
```
from the original paper, GMAE is unsupervised without label,  why the function return the true label information(`y`), Could you give some suggestions?

**Comment by RinneSz** [comment]
Yes, those labels are actually redundant and not used in the following training and testing. They can be safely removed without any problem. You can simply ignore them. I here preserve the labels for convenience if anyone wants to do some experiments on labeled data.

**Comment by yuxiaokang-source** [comment]
OK, thanks for the quick reply.
